backend/src/modules/quack/mutation.js

```diff
@@ -1,16 +0,0 @@
-export const addQuack = async (_, { userId, text }, { dbConnection }) => {
```

Comment:
máš tu někde mutaci na login/registraci uživatele?

Reply:
Hele to nemám bohužel. Je něco připravený v tom projektu, al eto asi moc použittelný nebude.